File: test/fsharpTests/IntTest/divideTest.fs
testCase "divide(3,2)" 
<| fun _ -> 
    let expected = 1
    Expect.equal expected (Int.divide 3 2) "error"
testCase "divide(3,0)" 
<| fun _ -> 
    Expect.equal (Int.divide 3 0) |> failwith "error"
testCase "divide(27,5)" 
<| fun _ -> 
    let expected = 5
    Expect.equal expected (Int.divide 27 5) "error"

File: test/ocamlTests/IntTest/divideTest.ml
test "divide(3,2)" (fun () -> expect (Int.divide 3 2) |> toEqual Eq.int 1) ; 
test "divide(3,0)" (fun () -> expect (fun () -> divide 3 0) |> toThrow); 
test "divide(27,5)" (fun () -> expect (Int.divide 27 5) |> toEqual Eq.int 5) ; 
